ndt7-client-cc.cpp

```diff
@@ -85,6 +85,24 @@ void BatchClient::summary() noexcept {
   std::cout << summary.dump() << std::endl;
 }
 
+struct KV {
+  std::string first;
+  std::string last;
```

Comment:
```suggestion
  std::string rest;
```

This seems to point to the rest of the string rather than to the last parameter.